include/mbedtls/check_config.h

```diff
@@ -789,7 +798,7 @@
 #endif
 
 #if defined(MBEDTLS_SSL_TICKET_C) && !defined(MBEDTLS_CIPHER_C)
-#error "MBEDTLS_SSL_TICKET_C defined, but not all prerequisites"
+#error "MBEDTLS_SSL_TICKET_C defined, but not all prerequisites (missing CIPHER)"
```

Comment:
Actually, after the recent work, I think this depends only on `CIPHER_C || USE_PSA_CRYPTO`.